tools/tcpconnect.py

```diff
@@ -27,11 +28,15 @@
 from socket import inet_ntop, ntohs, AF_INET, AF_INET6
 from struct import pack
 from time import sleep
+import dnslib
+from expiringdict import ExpiringDict
+from datetime import datetime
 
 # arguments
 examples = """examples:
     ./tcpconnect           # trace all TCP connect()s
     ./tcpconnect -t        # include timestamps
+    ./tcpconnect -d        # include DNS queries associated with connects
```

Comment:
Hmm. I try to keep these tools simple -- do one job and do it well -- and this is adding UDP tracing to a "tcpconnect" tool, which seems like a violation. But it'll also be a common real world use case, so maybe it is ok in this case.